readme.md

```diff
@@ -96,7 +96,7 @@ const prompts = require('prompts');
 
 ### Prompt Chain
 
-Prompt with a list of prompt objects. Returns object with response.
+Prompt with a list of prompt objects. Returns object with a response.
```

Comment:
Please revert this change change – or maybe change `a response` to `responses`? What do you think @DrunkenPoney @koppor?

Reply:
The number of responses goes with the number of prompt objects specified. If more than one object is provided then there would be more than one responses. So I think I'd say either "_Returns **an** object with one or more responses_" or just "_Returns **an** object with the responses_".